WindowsServerDocs/manage/windows-admin-center/azure/azure-extended-network.md

```diff
@@ -0,0 +1,185 @@
+
+### Configuration in Azure
+
+Before you use Windows Admin Center, you must perform the following steps through the Azure Portal:
+
+1. Create a Virtual network in Azure that contains at least two subnets, in addition to subnets required for your gateway connection.  One of the subnets you create must use the same subnet CIDR as the on-premises subnet you want to extend.  The subnet must be unique within your routing domain so that it does not overlap with any on-premises subnets.
+2. Configure a Virtual Network Gateway to use a Site-to-site or Express Route connection to connect the Azure Virtual network to your on-premises network.  
+3. Create a Windows Server 2019 VM in Azure that is capable of running nested virtualization.  Connect the primary network interface to the routable subnet, and the second network interface to the extended subnet.  
+4. Start the VM, enable the Hyper-V role, and reboot.  For example:
+
+```powershell
+Install-WindowsFeature -Name Hyper-V -IncludeManagementTools -Restart
+```
+
+5. Create two external Virtual Switches in the VM and connect one to each of the network interfaces.  For example:
+
+```powershell
+New-VMSwitch -Name "External" -AllowManagementOS $true -NetAdapterName "Ethernet"
+New-VMSwitch -Name "Extended" -AllowManagementOS $true -NetAdapterName "Ethernet 2"
+```
+
+### On-premises configuration
+
+You must also perform some manual configuration in your on-premises infrastructure:
+
+1. Make sure the subnet you want to extend and a second subnet that is unique, and does not overlap with any subnets in the Azure Virtual Network, are available on the physical machine where you will deploy the on-premises VM.
+2. Create a Windows Server 2019 VM on any hypervisor that supports nested virtualization.   It is strongly recommended that you create this as a highly available VM in a cluster.  Connect a virtual network adapter to the routable subnet, and a second virtual network adapter to the extended subnet.  
+3. Start the VM, enable the Hyper-V role, and reboot.
+
+```powershell
+Install-WindowsFeature -Name Hyper-V -IncludeManagementTools -Restart
+```
+
+4. Create two external Virtual Switches in the VM and connect one to each of the network interfaces.
+
+```powershell
+New-VMSwitch -Name "External" -AllowManagementOS $true -NetAdapterName "Ethernet"
+New-VMSwitch -Name "Extended" -AllowManagementOS $true -NetAdapterName "Ethernet 2"
+```
+
+5. You will need a separate machine for running the Windows Admin Center gateway.  You can create another Windows Server VM for this purpose, or any Windows 10 PC.
+
+### Additional pre-requisites
+
+If you have a firewall in between your on-premises network and Azure, you must configure it to allow for asymmetric routing.  This may include steps to disable sequence number randomization and enabling TCP state bypass. Refer to your firewall vendor documentation for these steps.
```

Comment:
```suggestion
If you have a firewall between your on-premises network and Azure, you must configure it to allow for asymmetric routing. This may include steps to disable sequence number randomization and enabling TCP state bypass. Refer to your firewall vendor documentation for these steps.
```